ga4gh/backends.py

```diff
@@ -280,30 +280,30 @@ def __init__(self, variantSetId, vcfPath):
         # self.tb chrom -> vcf dict
         self._tabixMap = {}
 
-        for fn in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
-            tabixfile = pysam.Tabixfile(fn)
+        for vcfFile in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
+            tabixfile = pysam.Tabixfile(vcfFile)
             for chrom in tabixfile.contigs:
                 if chrom in self._tabixMap:
                     raise Exception("cannot have overlapping VCF files.")
                 self._tabixMap[chrom] = tabixfile
 
     def convertVariant(self, record):
-        v = protocol.GAVariant()
```

Comment:
+1 I definitely agree.  For clarity, we can have it in both of the of the following locations, with an additional Readme file:

1) At the beginning of the file as in the following for defining reads, though it would pertinent to the `[server]` implementation:

https://github.com/ga4gh/schemas/blob/master/src/main/resources/avro/reads.avdl

2) Before each function definition if it is local to that scope.

I feel that each function could benefit with an additional function contract for better clarity.  This would entail something as follows, besides what is already written as a description.  If the functions are too complex, then we can modularize them:

```
 /*   
  *   convertVariant: ( self, record ) -> GAVariant
  *
  *   Definition: It will create a GAVariant from information related in
  *               self, and the variant and call information in the
  *               record.  GAVariant will also store all the relevant
  *               GACall information associated to it, that is extracted
  *               from record.
  *    
  *   Inputs:
  *       self   : is a ...
  *       record : is a ...
  *    
  *   Output: GAVariant
  *
  *   Returns: A populated GAVariant object.
  */
```
 ~p